GoedeDoelenHelpen/ClientApp/src/app/IsAuthenticated.guard.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, RouterStateSnapshot, ActivatedRouteSnapshot, Router } from '@angular/router';
+import { Observable } from 'rxjs/Observable';
+import { AuthenticationService } from './authentication.service';
+import { tap } from 'rxjs/operators';
+
+@Injectable()
+export class IsAuthenticated implements CanActivate {
+
+    constructor(
+        private authenticationService: AuthenticationService,
+        private router: Router
+    ) {}
+
+    canActivate(
```

Comment:
Looks like this is simply canActivate instead of the implementation of CanActivate?